scripts/docker-entrypoint.sh

```diff
@@ -126,6 +129,25 @@ else
    export SECRET_CASSANDRA_PASSWORD=${CASSANDRA_PASSWORD}
 fi
 
+if [ -e "$hazelcast_username_file" ]; then
+   export SECRET_HZ_CS_AUTH_USERNAME=$(<${hazelcast_username_file})
+else
+   export SECRET_HZ_CS_AUTH_USERNAME=${HZ_CS_AUTH_USERNAME}
+fi
+
+if [ -e "$hazelcast_password_file" ]; then
+   export SECRET_HZ_CS_AUTH_PASSWORD=$(<${hazelcast_password_file})
+else
+   export SECRET_HZ_CS_AUTH_PASSWORD=${HZ_CS_AUTH_PASSWORD}
+fi
+
+if [ "HZ_CLIENT_MODE" = true ]; then
```

Comment:
Don't we need `==` here

Reply:
I think both of them will work, we tested this with the variable with false and true values  and it worked well, but to be consistent changed it to "==" and pushed changes.